src/Polysemy/Internal.hs

```diff
@@ -129,6 +131,30 @@ newtype Sem r a = Sem
   }
 
 
+------------------------------------------------------------------------------
+-- | Makes constraints of functions that use multiple effects more readable by
+-- translating single list of effects into multiple 'Member' constraints:
+--
+-- @
+-- foo :: 'Members' \'[ 'Polysemy.Output.Output' Int
+--                 , 'Polysemy.Output.Output' Bool
+--                 ] r
```

Comment:
This might be a little clearer if it uses an effect that isn't the same between each case. Maybe two `Output`s and a `State`?

Reply:
Done.